_ml-commons-plugin/api.md

```diff
@@ -156,13 +156,22 @@ The response contains the following model information:
 
 ## Registering a model
 
-Before you register a model, you must [register a model group]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#registering-a-model-group) for the model.
-{: .important} 
+All versions of a particular model are held in a model group. You can [register a model group]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#registering-a-model-group) first, before registering a model to the model group. Each model group name must be globally unique in the cluster. 
 
-All versions of a particular model are held in a model group. After you register a model group, you can register a model to the model group. ML Commons splits the model into smaller chunks and saves those chunks in the model's index.
+If you are registering the first version of the model and you don't register a model group before registering the model, a new model group will be created automatically with the following name and access level:
+
+- Name: The new model group will have the same name as the model. Because the model group name must be unique, ensure that your model name does not have the same name as any model groups in the cluster. 
```

Comment:
```suggestion
- Name: The new model group has the same name as the model. Because the model group name must be unique, ensure that your model name is not the same as any other model group name in the cluster. 
```

Reply:
In this case, I think the future tense makes it clearer. "Other" does not apply here because one is the model and the others are model groups.